### PR TITLE
update the scope of the OAuthTocken for GitHub

### PR DESCRIPTION
**Pull request by rbaumgar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,6 +155,7 @@ spec:
 #### Authenticating to GitHub
 
 Authentication to GitHub can be performed using an [OAuth Personal Access Token](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token) or as an [GitHub App](https://docs.github.com/en/developers/apps/getting-started-with-apps/about-apps#about-github-apps), using a secret key and appId.
+The OAuth Personal Access Token needs to scope of admin:org/read:org.
```
Comment (sabre1041) [inline README.md:158]:
```suggestion
The OAuth Personal Access Token needs to scope of `admin:org/read:org`.
```

**Review by sabre1041 — CHANGES_REQUESTED** [review]
Minor formatting change.

File: README.md (hunk 1 above)
Comment (rbaumgar) [reply README.md:158]: thx!

**Comment by sabre1041** [comment]
@rbaumgar any chance you might be able to address the requested changes?

**Comment by rbaumgar** [comment]
done. please review

**Review by sabre1041 — APPROVED** [review]
LGTM

**Comment by rbaumgar** [comment]
thx

**Pull request merged**